CarInspection/src/WebCarInspection/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,51 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>@ViewData["Title"] - Car inspectios</title>
+    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="~/lib/bootstrap-datetimepicker/css/bootstrap-datetimepicker.min.css" />
+    <link rel="stylesheet" href="~/css/site.css" />
+</head>
+<body>
+    <header>
+        <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+            <a class="navbar-brand" href="#">Car inspection</a>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav mr-auto">
+                    <li class="nav-item active">
+                        <a class="nav-link" href="/Drivers/ShowDrivers">Drivers</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/Inspections/ShowInspections">Inspections</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/Inspectors/ShowInspectors">Inspectors</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/Violations/ShowViolations">Violations</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/Violators/ShowViolators">Violators</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <div class="container">
+        <main role="main" class="pb-3">
+            @RenderBody()
+        </main>
+    </div>
+
+    <script src="~/lib/jquery/jquery.min.js"></script>
+    <script src="~/lib/jquery-validate/jquery.validate.min.js"></script>
```

Comment:
jquery-validate already described in  _validationscriptspartial.cshtml.
You should just render this file in views that require validation.
Please remove this two scripts from this file.